SocialCareCaseViewerApi/V1/Infrastructure/Person.cs

```diff
@@ -54,7 +65,32 @@ public class Person : IAuditEntity
 
         [Column("first_language")]
         [MaxLength(100)]
-        public string FirstLanguage { get; set; }
+        public string? FirstLanguage { get; set; }
+
+        [Column("fluent_in_english")]
```

Comment:
double-checking: are these new keys camel-case or snake-case in the api response?